app/views/rsvp_mailer/send_alert_email.html.erb

```diff
@@ -16,7 +16,7 @@
             <%end%>
         </div>
       <%end%>
-      <p><b>Message: </b><%= guest.family.message %></p>
```

Comment:
This works too, but you could just change this to `@guest.family.message` and not need to add anything to `rsvp_mailer.rb`